types/rpc_id.go

```diff
@@ -0,0 +1,81 @@
+package types
```

Comment:
Thoughts on renaming this file to `rpc_helpers`, `rpc_default` or `rpc_errors`?

Probably the last one is best.